cmd/ndt7-client/main.go

```diff
@@ -184,10 +184,10 @@ func main() {
 	}
 
 	// If a service URL is given, then only one direction is possible.
-	if flagService.URL != nil && strings.Contains(flagService.URL.Path, params.DownloadURLPath) {
+	if flagService.URL != nil && strings.HasSuffix(flagService.URL.Path, params.DownloadURLPath) {
```

Comment:
This is a change, I'm happy to revert this back to `Contains`, and sync `ndt7.go` along with it.

I figured it might be nice if the logic in both places was similar.

Reply:
Note that this also prevents something like:

```
ndt7-client -service-url 'wss://speedtest.example.com/ndt/v7/upload/ndt/v7/download`
```